### PR TITLE
[Priest] Add sins of the many

### PR DESCRIPTION
**Pull request by taherbert**

### CODE REVIEW

File: engine/class_modules/priest/sc_priest.hpp [hunk 1]
```diff
@@ -728,6 +729,10 @@ struct priest_action_t : public Base, public parse_buff_effects_t<priest_td_t>
     parse_buff_effects( p().buffs.words_of_the_pious );       // Spell Direct amount for Smite and Holy Nova
     parse_buff_effects( p().buffs.gathering_shadows, true );  // Spell Direct amount for Mind Sear (NOT DP)
     parse_buff_effects( p().buffs.devoured_pride );           // Spell Direct and Periodic amount
+
+    // Discipline
+    parse_buff_effects( p().buffs.shadow_covenant );
+    parse_buff_effects( p().buffs.sins_of_the_many, false, true );
```
Comment (seanpeters86) [inline engine/class_modules/priest/sc_priest.hpp]: Can you add a comment like the spells above for what exactly this is "automagically" applying?

File: engine/class_modules/priest/sc_priest_discipline.cpp [hunk 2]
```diff
@@ -280,6 +280,12 @@ void priest_t::create_buffs_discipline()
   buffs.shadow_covenant =
       make_buff( this, "shadow_covenant", talents.discipline.shadow_covenant->effectN( 4 ).trigger() )
           ->set_trigger_spell( talents.discipline.shadow_covenant );
+
+  // 280391 has the correct 40% damage increase value, but does not apply it to any spells.
+  // 280398 applies the damage to the correct spells, but does not contain the correct value (12% instead of 40%).
+  // That 12% represents the damage REDUCTION taken from the 40% buff for each attonement that has been applied.
```
Comment (seanpeters86) [inline engine/class_modules/priest/sc_priest_discipline.cpp:286]: Maybe note as a `TODO` to add support for Atonements reducing this?

**Review by seanpeters86 — CHANGES_REQUESTED** [review]

File: engine/class_modules/priest/sc_priest.hpp (hunk 1 above)
Comment (taherbert) [reply engine/class_modules/priest/sc_priest.hpp]: Done

File: engine/class_modules/priest/sc_priest_discipline.cpp (hunk 2 above)
Comment (taherbert) [reply engine/class_modules/priest/sc_priest_discipline.cpp:286]: Done

**Review by seanpeters86 — APPROVED** [review]

**Pull request merged**